index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="./style.css">
+        <title>Mobile Version</title>
+    </head>
+
+    <body>
+        <main class="mobile">
+            <header class="toolbar">
+                <h2 class="john">John Doe</h2>
```

Comment:
- [ ] Kindly wrap this in an `a` tag. Logos are meant to work as links to the homepage.

Reply:
@JuliCarracedo What should I wrap in an <a> tag?

Reply:
`<a><h2 class="john">John Doe</h2></a>`